tests/components/deconz/test_binary_sensor.py

```diff
@@ -106,8 +96,7 @@ async def test_binary_sensors(hass):
     gateway.api.event_handler(state_changed_event)
```

Comment:
This is also an example where we're interacting with integration code in the tests.

It would be better to patch the library api in a fixture that we can have available in the test so we can call the event handler via the patched api fixture.

A comment for the future.